.vscode/settings.json

```diff
@@ -23,5 +23,6 @@
   "git.ignoreLimitWarning": true,
   "cSpell.words": [
     "datalake"
-  ]
+  ],
+  "prettier.prettierPath": "./sdk/core/core-rest-pipeline/node_modules/prettier"
```

Comment:
Should we ping against `./common/temp/node_modules/.bin/prettier` instead?

Reply:
Not sure if it needs the root directory for the package vs the bin alias. Also not sure how stable the common/temp pattern is given it's an implementation detail of rush.

@jeremymeng any thoughts?

Reply:
tbh I am not sure either. I am mainly worried whether this works in packages that don't depend on core-rest-pipeline, including messaging and some other core libraries. Will core-rest-pipeline's `/sdk/core/core-rest-pipeline/node_modules/prettier` always exist after running `rush update`?

Reply:
A generic rush update should work since that links all node_modules folders. You shouldn't have to actually build core-rest-pipeline.

Reply:
prettier is a dev dependency of core-rest-pipeline so it's expected to find perttier under core-rest-pipeline/node_modules/prettier I believe, so this should be pretty stable unless we move/rename any part of the path.